translate/cloud-client/translate_v3_translate_text.py

```diff
@@ -16,7 +16,7 @@
 from google.cloud import translate
 
 
-def sample_translate_text(project_id="YOUR_PROJECT_ID"):
+def sample_translate_text(text="YOUR_TEXT_TO_TRANSLATE", project_id="YOUR_PROJECT_ID"):
```

Comment:
my only question is are we no using comment blocks for variables and defining arguments now

Reply:
For last few PRs, reviewers did not approve that so I changed it to current style.

Reply:
Yea, we switched over to use defaults rather than comment blocks.